azure-pipelines.yml

```diff
@@ -34,6 +34,8 @@ jobs:
           BUILDER: 'AMD64 Conda Python 3.6'
         AMD64 Conda Python 3.7:
           BUILDER: 'AMD64 Conda Python 3.7'
+        AMD64 Conda Python 3.8:
```

Comment:
I wasn't sure if this was omitted for some reason, and I wasn't sure if this was a "min/max". I figured I probably shouldn't be adding 3.9 here, but I can adjust as needed.